kernel/src/common/utils.rs

```diff
@@ -15,7 +15,7 @@
 /// destructor.
 #[macro_export]
 macro_rules! static_init {
-    ($T:ty, $e:expr) => {{
+    ($T:ty, $e:expr $(,)?) => {{
```

Comment:
I understand the cases where there are lists, but why would you want to add a comma after the second argument to something like static_init?